src/__test__/utils/builders.ts

```diff
@@ -76,30 +79,48 @@ export const buildFirmwareConstants = (...overrides: any) => {
   } as FirmwareConstants;
 };
 
-export const buildGetAddressesObject = ({ ...overrides }) => ({
+export const buildWallet = (overrides?) => ({
```

Comment:
Probably better to call this something like `buildTestWallet`

Reply:
Meh, I'm going through this file and now realizing that we don't add `test` to any of these functions, so I guess you can leave it